src/standard_headers.inc

```diff
@@ -0,0 +1 @@
+static const std::vector<std::string> kStandardHeaders = { "algorithm", "any", "array", "atomic", "bitset", "cassert", "ccomplex", "cctype", "cerrno", "cfenv", "cfloat", "chrono", "cinttypes", "ciso646", "climits", "clocale", "cmath", "codecvt", "complex", "complex.h", "condition_variable", "coroutine", "csetjmp", "csignal", "cstdarg", "cstdbool", "cstddef", "cstdint", "cstdio", "cstdlib", "cstring", "ctgmath", "ctime", "ctype.h", "cwchar", "cwctype", "cxxabi.h", "deque", "dynarray", "errno.h", "exception", "experimental", "ext", "filesystem", "float.h", "forward_list", "fstream", "functional", "future", "initializer_list", "inttypes.h", "iomanip", "ios", "iosfwd", "iostream", "istream", "iterator", "limits", "limits.h", "list", "locale", "locale.h", "map", "math.h", "memory", "memory_resource", "module.modulemap", "mutex", "new", "numeric", "optional", "ostream", "propagate_const", "queue", "random", "ratio", "regex", "scoped_allocator", "set", "setjmp.h", "shared_mutex", "sstream", "stack", "stdbool.h", "stddef.h", "stdexcept", "stdint.h", "stdio.h", "stdlib.h", "streambuf", "string", "string.h", "string_view", "strstream", "support", "system_error", "tgmath.h", "thread", "tuple", "typeindex", "typeinfo", "type_traits", "unordered_map", "unordered_set", "utility", "valarray", "variant", "vector", "wchar.h", "wctype.h" };
```

Comment:
This list made me sad. I think the header completion can be achieved with either

```emacs
;; doom-emacs
  (set! :company-backend '(c-mode c++-mode) '(company-lsp company-c-headers))
```

or

```emacs
(setq ccls-extra-init-params '(:clang (:extraArgs ["-cxx-isystem", "/usr/include/c++/v1", "-c-isystem", "/usr/include"])))
```

Reply:
I also tend to avoid duplicating cquery's logic of "detecting system include paths with an external program"

Reply:
The latter solution seems to be OK. It's handy to provide basic system header completion out-of-box.